tex/gregoriotex-main.tex

```diff
@@ -1117,10 +1067,10 @@
 
 %macro called at the beginning of a score
 % #1 is the gabc score id
-% #2 is the high height
-% #3 is the low height
-% #4 is 1 if there is a translation somewhere
-% #5 is if 1 if we have space above the staff
+% #2 is the high height (currently ignored)
```

Comment:
Can you foresee a future use for these arguments?  If not, I'd mark them as deprecated, not just ignored.  Once this PR is approved and merged, then we can work on the appropriate PR for the develop branch which will actually remove them (since they'd have to be removed from the executable, that's not a ctan eligible change).

Reply:
I don't foresee a future use, no. How do I mark them as deprecated?

Reply:
I've updated the Wiki page on the Deprecation process to cover this case: https://github.com/gregorio-project/gregorio/wiki/Deprecation-Process